.github/workflows/cli-coverage.yml

```diff
@@ -0,0 +1,40 @@
+name: test coverage
+
+on:
+  push:
+    branches: [master, dev]
+
+jobs:
+  cli-coverage-report:
+    strategy:
+      matrix:
+        python-version: [ "3.11" ]
```

Comment:
Other verrsions? I know we flank things like test python 3.8 and 3.11...

Reply:
It's a coverage test, so we don't need other versions